contracts/MechMarketplace.sol

```diff
@@ -438,7 +447,7 @@ contract MechMarketplace is IErrorsMarketplace {
             revert ZeroAddress();
         }
 
-        // Check that the request is not already delivered
+        // Check if request has been delivered
         if (mechDelivery.deliveryMech != address(0)) {
             revert AlreadyDelivered(requestId);
         }
```

Comment:
Currently this never happens, but let's leave it here for possible mechs change.